example/spec/parallel/testAPI/snsRuleSpec.js

```diff
@@ -64,6 +64,17 @@ describe('The SNS-type rule', () => {
 
   afterAll(async () => {
     await SNS.deleteTopic({ TopicArn: snsTopicArn }).promise();
+
```

Comment:
The changes in this file are unrelated to this ticket, but Menno and I found that this test was not always cleaning up after itself, and would never pass afterward unless manual intervention was taken.  Made the test more robust, and also cleans up after itself now.

Reply:
💯